include/SFML/Window/Clipboard.hpp

```diff
@@ -0,0 +1,108 @@
+////////////////////////////////////////////////////////////
+//
+// SFML - Simple and Fast Multimedia Library
+// Copyright (C) 2007-2017 Laurent Gomila (laurent@sfml-dev.org)
+//
+// This software is provided 'as-is', without any express or implied warranty.
+// In no event will the authors be held liable for any damages arising from the use of this software.
+//
+// Permission is granted to anyone to use this software for any purpose,
+// including commercial applications, and to alter it and redistribute it freely,
+// subject to the following restrictions:
+//
+// 1. The origin of this software must not be misrepresented;
+//    you must not claim that you wrote the original software.
+//    If you use this software in a product, an acknowledgment
+//    in the product documentation would be appreciated but is not required.
+//
+// 2. Altered source versions must be plainly marked as such,
+//    and must not be misrepresented as being the original software.
+//
+// 3. This notice may not be removed or altered from any source distribution.
+//
+////////////////////////////////////////////////////////////
+
+#ifndef SFML_CLIPBOARD_HPP
+#define SFML_CLIPBOARD_HPP
+
+////////////////////////////////////////////////////////////
+// Headers
+////////////////////////////////////////////////////////////
+#include <SFML/Window/Export.hpp>
+#include <SFML/System/String.hpp>
+
+
+namespace sf
+{
+////////////////////////////////////////////////////////////
+/// \brief Give access to the system clipboard
+///
+////////////////////////////////////////////////////////////
+class SFML_WINDOW_API Clipboard
+{
+public:
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Get the content of the clipboard as string data
+    ///
+    /// This function returns the content of the clipboard
+    /// as a string. If the clipboard does not contain string
+    /// it returns an empty sf::String object.
+    ///
+    /// \return Clipboard contents as sf::String object
+    ///
+    ////////////////////////////////////////////////////////////
+    static String getString();
+
+    ////////////////////////////////////////////////////////////
+    /// \brief Set the content of the clipboard as string data
+    ///
+    /// This function sets the content of the clipboard as a
+    /// string.
+    ///
+    /// \param sf::String containing the data to be sent 
```

Comment:
While working on the OS X impl, I noticed this warning:

```
In file included from SFML/src/SFML/Window/Clipboard.cpp:28:
SFML/include/SFML/Window/Clipboard.hpp:63:16: warning: parameter 'sf::String' not found in the function declaration [-Wdocumentation]
    /// \param sf::String containing the data to be sent 
               ^~~~~~~~~~
SFML/include/SFML/Window/Clipboard.hpp:63:16: note: did you mean 'text'?
    /// \param sf::String containing the data to be sent 
               ^~~~~~~~~~
               text
1 warning generated.
```

Could you add `text` (the param name) after `\param`? Thanks.